doc/build.md

```diff
@@ -209,6 +209,32 @@ Or, you can copy over the .so to `/usr/local/lib` or something similar.
 
 One this is done, you should be able to run `cargo build --all`
 
+## Building docker images
+
+Use NIX to build the images. Note that the images are based on NIX packages
+that are built as part of building the image. The tag of the image will be
+short commit hash of the top-most commit or a tag name of the commit if
+present. Example of building a moac package:
+
+```bash
+nix-build -A images.moac-image
+```
+
+At the end of the build is printed path to docker image tar archive. Import
+it to a docker (don't use _import_ command) and run bash to poke around:
```

Comment:
Perhaps rephrase as "Load the image into Docker".

Reply:
done. thanks! 👍